src/App.js

```diff
@@ -1,25 +1,17 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import { AppRouting } from './components/routing/AppRouting';
+import { BrowserRouter as Router } from 'react-router-dom';
+import Navigation from './components/routing/Navigation';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Router>
+      <div className="App">
+        <AppRouting />
+      </div>
+      <Navigation />
+    </Router>
```

Comment:
Lines 9-10: They are so easier to read and maintain, and have good naming components. I can understand quickly what you want to do with those lines. Awesome! 🥳